homeassistant/package_constraints.txt

```diff
@@ -68,7 +68,7 @@ standard-aifc==3.13.0
 standard-telnetlib==3.13.0
 typing-extensions>=4.14.0,<5.0
 ulid-transform==1.4.0
-urllib3>=1.26.5,<2
+urllib3>=1.26.5
```

Comment:
Should we update the constraint to `>=2.0` or even something newer like `2.2` or `2.4`? That way we could prevent accidentally installing an old version again.

_Note that it might cause issues with custom components but maybe the best way to deal with those would be to see what breaks after the release and fix the dependencies then accordingly._